modules/olm-installing-from-operatorhub-using-cli.adoc

```diff
@@ -163,3 +163,7 @@ $ oc apply -f sub.yaml
 ----
 +
 At this point, OLM is now aware of the selected Operator. A cluster service version (CSV) for the Operator should appear in the target namespace, and APIs provided by the Operator should be available for creation.
+
+ifeval::["{context}" == "olm-installing-operators-in-namespace"]
+:!olm-user:
+endif::[]
```

Comment:
I can't quite figure out what this is doing - I guess it specifically is for the OLM docs which I don't know anything about, but I trust you :)

Reply:
Ah yes, sorry, meant to explain in description: while trying to troubleshooting the earlier symlink failures, I saw that this file was setting `:olm-user:` at the top, but was not unsetting at the bottom, so this is just tidying that up.